packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -32,6 +33,8 @@ const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 type Properties = ChatProperties & {
   title: string;
   showDayHeaders: boolean;
+  dayHeaderFormat: Format;
```

Comment:
It's better to consider that these properties are optional as it will in public d.ts file

Reply:
https://github.com/DevExpress/DevExtreme/pull/28234/commits/146df9b953cfca2760258090346ce0e5f1173871